packages/ai-jsx/src/lib/tts/tts.tsx

```diff
@@ -0,0 +1,360 @@
+'use client';
+
+/**
+ * Defines a function that can be used to build a URL for a text-to-speech
+ * service. This URL will be used to retrieve an audio file that can be
+ * played by an HTML5 audio element.
+ */
+export type BuildUrlFunction = (provider: string, voice: string, rate: number, text: string) => string;
```

Comment:
```suggestion
export type BuildUrl = (provider: string, voice: string, rate: number, text: string) => string;
```

minor, but typically you don't include the name of the superclass in the type. e.g. you wouldn't call it `PersonObject`; just `Person`.